common.sh

```diff
@@ -127,3 +127,9 @@ LOGFILE="$LOGDIR/$(basename $0 .sh)-$(date +%F-%H%M%S).log"
 echo "Logging to $LOGFILE"
 # Set fd 1 and 2 to write to the log file
 exec 1> >( tee "${LOGFILE}" ) 2>&1
+
+# Set up virtual infrastructure with dev-scripts. Default: yes
+OOOQ_SETUP=${OOOQ_SETUP:-y}
```

Comment:
If we can't reuse NODES_PLATFORM for this we should avoid the OOOQ name IMO - it makes sense to those of us who know where the ansible roles originated, but it's probably confusing to anyone that didn't previously work with OOOQ :)

Reply:
I agree, I'm going to refactor around using the NODES_PLATFORM option.